README.md

```diff
@@ -27,7 +27,45 @@ add the code snippet from https://github.com/SWAuk/plg_swa_viewlevels
 The original template sources is https://www.favthemes.com/joomla-templates/product/favourite-free-responsive-joomla-3-template
 A copy of this can be found on the SWA Drive "Tech/Web Dev/Templates/favourite_j!3_UnzipFirst.zip"
 
-## Development Environment
+## Development - Dependencies
+
+### composer
+
+In order to install the needed packages for development you'll need [composer](https://getcomposer.org/download/).
+Composer is a dependency manager and command runner for PHP.
+
+Use it to install the PHP libraries needed for testing, development and production.
+
+As docker is already required, the easiest way to use `composer` to install the required libs is via the official [docker image](https://hub.docker.com/_/composer).
+
+You can also run **composer in docker**...
+
+For linux and bashy systems that would be:
+
+```sh
+docker run --rm -it -v $PWD:/app composer <command here>
+```
+
+If you are on windows using CMD that would be:
+
+```sh
+docker run --rm -it -v %CD%:/app composer <command here>
+```
+
+### docker & docker-compose
+
+If you want to run composer in docker, or use the recommended development environment then you need these installed.
+
+* [Docker](https://docs.docker.com/install/)
+* [docker-compose](https://docs.docker.com/compose/install/)
```

Comment:
Move this section above the composer section? Then it talks about installing docker before using docker.

Reply:
Will do